rules.tf

```diff
@@ -0,0 +1,125 @@
+locals {
+  ingress = "ingress"
+  egress = "egress"
```

Comment:
These, for consistency, I left them like this, since I had some that were repeated. I applied the rest of the naming suggestions. I hope it is fine with you